content/docs/introduction/monitoring-page.md

```diff
@@ -78,54 +84,60 @@ Idle connections are those that are open but not currently being used. While a f
 
-The **Replication delay seconds** graph shows the time delay, in seconds, between the last transaction committed on the primary compute and the application of that transaction on the replica. A higher value suggests that the replica is behind the primary, potentially due to network latency, high replication load, or resource constraints on the replica. This chart is only visible when selecting a **Replica** compute from the **Compute** drop-down menu.
+### Replication delay seconds
 
 ![Replication delay seconds](/docs/introduction/rep_delay_seconds.png)
+
+The **Replication delay seconds** graph shows the time delay, in seconds, between the last transaction committed on the primary compute and the application of that transaction on the replica. A higher value suggests that the replica is behind the primary, potentially due to network latency, high replication load, or resource constraints on the replica. This graph is only visible when selecting a **Replica** compute from the **Compute** drop-down menu.
+
+### Local file cache hit rate
+
+![local file cache hit rate graph](/docs/introduction/local_file_cache_hit_rate.png)
+
+The **Local file cache hit rate** graph shows the percentage of read requests served from memory &#8212; from Neon's Local File Cache (LFC). 
+Queries not served from either Postgres shared buffers (128 MB on all Neon computes) or the Local File Cache retrieve data from storage, which is more costly and can result in slower query performance. To learn more about how Neon caches data and how the LFC works with Postgres shared buffers, see [What is the Local File Cache?](/docs/extensions/neon#what-is-the-local-file-cache)
+
+### Working set size
+
+![working set size graph](/docs/introduction/working_set_size.png)
+
+Your working set is recently accessed data and indexes stored in memory&#8212;in Neon's [Local File Cache (LFC)](/docs/extensions/neon#what-is-the-local-file-cache) for quick access.
+
+The **Working set size** graph provides a visual representation of how much data is being accessed from the LFC over different time intervals. Here’s how to interpret the graph:
+
+- **5m** (5 minutes): This line shows how much data has been accessed from the cache in the last 5 minutes.
+- **15m** (15 minutes): Similar to the 5-minute window, this metric tracks data accessed from the cache over the last 15 minutes.
+- **1h** (1 hour): This line represents the amount of data accessed from the cache in the last hour.
+- **Local file cache size**: This is the size of the the LFC, which is determined by the size of your compute. Larger computes have larger caches. For cache sizes, see [How to size your compute](/docs/manage/endpoints#how-to-size-your-compute).
```

Comment:
```suggestion
- **Local file cache size**: This is the size of the the LFC, which is determined by the size of your compute. Larger computes have larger caches. For cache sizes, see [How to size your compute](/docs/manage/endpoints#how-to-size-your-compute).
For optimal performance the local file cache should be larger than your working set size for a given time interval.
If your working set size is larger than the LFC size it is recommended to increase the maximum size of the compute to improve the LFC hit rate and achieve good performance.

If your workload pattern doesn't change much over time it is recommended to compare the 1h time interval working set size with the LFC size and make sure that working set size is smaller than LFC size.

Note that this recommendation only applies to workloads with a working set larger than 128 MiB, because workloads with a working set smaller than 128 MiB can be completely served out the Postgres shared buffer and the compute size is irrelevant in this case for the cache hit rate.
```